trin-core/src/cli.rs

```diff
@@ -180,18 +218,28 @@ impl TrinConfig {
         I: Iterator<Item = T>,
         T: Into<OsString> + Clone,
     {
-        let config = Self::from_iter(args);
+        let config = match Self::from_iter_safe(args) {
+            Ok(config) => config,
+            Err(error) => {
+                match error.kind {
+                    ErrorKind::ValueValidation => {
+                        println!("Fallback to default value, cause: {}", error);
+                    }
+                    _ => panic!("Could not create configuration with arguments"),
+                };
+                Default::default()
+            }
+        };
```

Comment:
Allow error handling when creating `TringConfig` from `structopt::StructOpt::from_iter(args)`. 
There are downsides to this:
- It will fallback to all default values, even if only one argument is wrong.
-  We should switch from using `println` to `tracing::{warn/error}`, but the error message from `clap::Error` uses Unicode for coloring, which can make it difficult to read.

I'm sure there might be a better solution.